### PR TITLE
Add CheckPangram in String

### PR DESCRIPTION
**Pull request by arcane2099**
# Welcome to JavaScript community

### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new JavaScript files are placed inside an existing directory.
* [x] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: String/CheckPangram.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Pangram is a uniques sentence that contains every letter of the alphabet
+// The alphabet has to appear at least once
+// Pangram Check is case sensitve
+function checkPangram (sentence) {
+  // Create an array of 26 characters
+  const alphabets = 'abcdefghijklmnopqrstuvwxyz'
```
Comment (ruppysuppy) [inline String/CheckPangram.js:6]:
```suggestion
  const ALPHABETS = 'abcdefghijklmnopqrstuvwxyz'
```
Move it out of the function body

Comment (arcane2099) [reply String/CheckPangram.js:6]: noted

File: String/CheckPangram.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 // Pangram is a uniques sentence that contains every letter of the alphabet
 // The alphabet has to appear at least once
 // Pangram Check is case sensitve
-function checkPangram (sentence) {
-  // Create an array of 26 characters
-  const alphabets = 'abcdefghijklmnopqrstuvwxyz'
 
+// Create an array of 26 characters
```
Comment (ruppysuppy) [inline String/CheckPangram.js:5]:
```suggestion
// Create a string with all 26 characters

```

**Comment by raklaptudirm** [comment]
`CheckPanagram.js` already exists.

**Pull request closed without merging**